test/e2e/storage/vsphere_volume_fstype.go

```diff
@@ -86,7 +86,7 @@ func invokeTestForFstype(f *framework.Framework, client clientset.Interface, nam
 	var pvclaims []*v1.PersistentVolumeClaim
 	pvclaims = append(pvclaims, pvclaim)
 	By("Waiting for claim to be in bound phase")
-	persistentvolumes, err := framework.WaitForPVClaimBoundPhase(client, pvclaims)
+	persistentvolumes, err := framework.WaitForPVClaimBoundPhase(client, pvclaims, 2*time.Minute)
```

Comment:
I don't like changing the timeout from 5 minutes to 2. Can you please use `framework.ClaimProvisionTimeout` everywhere and use 2 minutes only where you expect failure? Or create a new 2 min constant and use it everywhere.

Reply:
Fixed! I have used framework.ClaimProvisionTimeout everywhere and used 2 minutes only when I expect failure.